MainWindow/MainWindow.xaml.cs

```diff
@@ -1519,15 +1646,47 @@ private bool Read_AutoDetectID()
             // Assume failure
             bool result = false;
 
+            // Check if a device has been selected
+            if (pcapSelectedDevice == null)
+            {
+                uiDispatcher.Invoke((Action)(() =>
+                {
+                    Console.WriteLine("Please select a Pcap device from the Pcap dropdown.");
+                }));
```

Comment:
Console.WriteLine() by itself does not need to be called from the uiDispatcher; that is only needed when the UI-elements are changed directly. Recommend to remove the Invoke.

Reply:
Good to know.